geniesp/bpc_redcap_export_mapping.py

```diff
@@ -507,24 +511,24 @@ class BpcProjectRunner(metaclass=ABCMeta):
     # Sponsored project name
     _SPONSORED_PROJECT = ""
     # Redcap codes to cbioportal mapping synid and form key is in
-    # version 38 was the last stable version
-    # Use version 42 - but there is a bug in Synapse...
-    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.42"
+    # version 38, 42 were last stable version(s)
+    _REDCAP_TO_CBIOMAPPING_SYNID = "syn25712693.48"
```

Comment:
It's a bit irritating, because by specifying "syn25712693", it isn't actually using the latest version.  Lets say we created version 48 and then you update the tables without taking a snapshot (v 49).  pulling syn25712693 will grab whats in the table.

I wish querying on "syn25712693" would just do "syn25712693.{latest_version}"

Reply:
@thomasyu888 If we use [v49](https://www.synapse.org/#!Synapse:syn25712693.49/tables/), the latest version of the mapping file table, the only difference is that race_ethnicity is now colType STRING instead of NUMBER (from v42). I'm only saying this because I used v42 to create the test performance file which Brooke okay'd.

We need to use v49.